src/screens/d-apps/d-apps.tsx

```diff
@@ -59,7 +61,11 @@ export const DApps = () => {
   return (
     <>
       <InsetSubstitute type="top" />
-      <SearchInput placeholder="Search Dapp" onChangeText={setSearchQuery} />
+      <SearchInput
+        placeholder="Search Dapp"
+        onChangeText={setSearchQuery}
+        {...setTestID(DAppsSelectors.searchDAppsInput)}
```

Comment:
the same, you need to pass `testId` prop

Reply:
fixed